Joker/metro.config.js

```diff
@@ -4,14 +4,22 @@
  *
  * @format
  */
-
+const blacklist = require('metro-config/src/defaults/exclusionList');
 module.exports = {
   transformer: {
     getTransformOptions: async () => ({
       transform: {
         experimentalImportSupport: false,
-        inlineRequires: true,
+        inlineRequires: false,
```

Comment:
fixed it by analogy with the haqq branch storybook setup(otherwise it will cause problems later)